sixtools/plotting_functions.py

```diff
@@ -0,0 +1,105 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.patches import Rectangle
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+
+def plot_frame(ax, frame, light_ROIs=[], cax=None, **kwargs):
```

Comment:
To me this appears to be generic enough to be moved to scikit-beam/rixs. The input values match well with what is already in there. What are your thoughts?

Reply:
This is something of a marginal case, but scikit-beam/rixs/plotting_functions.py has stuff for plotting 3 column (x, y, I) data. i.e. the standard input to the package. plot_frame, on the other hand, acts upon a 2D array so I'm inclined to keep it here.